server/account/account.go

```diff
@@ -80,22 +80,22 @@ func (s *Server) UpdatePassword(ctx context.Context, q *account.UpdatePasswordRe
 	// Need to validate password complexity with regular expression
 	passwordPattern, err := s.settingsMgr.GetPasswordPattern()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to get password pattern: %w", err)
 	}
 
 	validPasswordRegexp, err := regexp.Compile(passwordPattern)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to compile password regex: %w", err)
 	}
 
 	if !validPasswordRegexp.Match([]byte(q.NewPassword)) {
 		err := fmt.Errorf("New password does not match the following expression: %s.", passwordPattern)
-		return nil, err
+		return nil, fmt.Errorf("failed to compile password regex: %w", err)
```

Comment:
The error is already wrapped on the line above. How about we just move the error from line 92 to line 93?

Reply:
@crenshaw-dev Got it, I'll commit again. 🥹